doc/manpages/faq.asciidoc

```diff
@@ -133,6 +133,16 @@ The shell scope allows users to spawn any interpreter they want, for a minimal
 cost in terms of performance, it is therefore the reason why it's the only
 one available by default.
 
+What shell is used to expand `%sh{}` scopes ?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The server expands shell scopes using `/bin/sh` (which is, in most
```

Comment:
Thats not strictly true, we go through the dirs specified by the `_CS_PATH` confstr (see `man confstr`) and find the first one that contains `sh`. This is *guaranteed* by posix to be a posix shell, its probably going to be /bin/sh on almost all distribution, but it seems that posix does not let us assume that `/bin/sh` is going to be a posix compliant shell.